core/trino-main/src/main/java/io/trino/sql/planner/ConnectorExpressionTranslator.java

```diff
@@ -235,6 +237,10 @@ public ConnectorToSqlExpressionTranslator(Session session, PlannerContext planne
                 }
             }
 
+            if (COALESCE_FUNCTION_NAME.equals(call.getFunctionName()) && call.getArguments().size() >= 2) {
```

Comment:
shouldn't coalesce be varargs?

Reply:
Coalesce expects at least two arguments at:
https://github.com/trinodb/trino/blob/master/core/trino-parser/src/main/java/io/trino/sql/tree/CoalesceExpression.java#L52